module/input/SensorFilter/src/SensorFilter.cpp

```diff
@@ -19,268 +19,133 @@
 
 #include "SensorFilter.hpp"
 
-#include "extension/Configuration.hpp"
-
-#include "message/actuation/BodySide.hpp"
-#include "message/input/Sensors.hpp"
-#include "message/platform/RawSensors.hpp"
+#include "Kinematics.hpp"
+#include "Odometry.hpp"
+#include "RawSensors.hpp"
```

Comment:
shouldn't need to include these, their declaration comes from the SensorFilter header file